config/default/manager_auth_proxy_patch.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: submariner-operator
```

Comment:
Whats this deployment in system namespace for?

Reply:
this name is just a placeholder and can be replaced when running kustomize. However, this yaml file is not included - see `config/default/kustomization.yaml`